doc/classes/RenderingServer.xml

```diff
@@ -2292,103 +2291,100 @@
 				To place in a scene, attach this reflection probe to an instance using [method instance_set_base] using the returned RID.
 			</description>
 		</method>
-		<method name="reflection_probe_set_as_interior">
+		<method name="reflection_probe_set_ambient_color">
 			<return type="void">
 			</return>
 			<argument index="0" name="probe" type="RID">
 			</argument>
-			<argument index="1" name="enable" type="bool">
+			<argument index="1" name="color" type="Color">
 			</argument>
 			<description>
-				If [code]true[/code], reflections will ignore sky contribution. Equivalent to [member ReflectionProbe.interior_enable].
```

Comment:
There might be some content that can be salvaged in here (CC @clayjohn @Calinou).